packages/sdks/mitosis.config.cjs

```diff
@@ -198,6 +198,73 @@ const filterActionAttrBindings = (json, item) => {
   });
 };
 
+const ANGULAR_PLUGIN = () => ({
+  code: {
+    post: (code) => {
+      const pathValue = code.match(/\[path\]="(.*?)"/);
+      if (pathValue) {
+        code = code.replace(
+          pathValue[0],
+          `[path]="${pathValue[1].replaceAll('`', "'").replaceAll('\\', '').replaceAll('.${', ".'+").replaceAll('}', "+'")}"`
+        );
+      }
+      return code;
+    },
+  },
+});
+
+
+const ANGULAR_PLUGIN2 = () => ({
+  code: {
+    post: (code) => {
+      if (code.includes('inputs: { getWrapperProps')) {
+        const wrapperObj = code.match(/inputs: {.*?}/s)[0].replace('inputs: {', '').replaceAll('props.', '') + ')';
+        const inputsObj = code.match(/inputs: {.*?;/s)[0].replace('inputs: ', '');
+        code = code.replace(inputsObj, wrapperObj.replace('context.value', 'context'));
+      }
+      return code;
+    }
+  }
+});
+
+const ANGULAR_PLUGIN3 = () => ({
+  code: {
+    post: (code) => {
+      if (code.includes('<enable-editor')) {
+        code = code.replace('[linkComponent]="linkComponent"', '');
+      }
+      if (code.includes('<block-wrapper')) {
+        code = code.replace('[linkComponent]="linkComponent"', '');
+      }
+      return code;
+    }
+  }
+});
+
+// for fixing circular dependencies
+const ANGULAR_PLUGIN4 = () => ({
+  code: {
+    post: (code) => {
+      if (code.includes('component-ref, ComponentRef') || code.includes('repeated-block, RepeatedBlock')) {
+        code = code.replace('imports: [CommonModule, Block]', 'imports: [CommonModule, forwardRef(() => Block)]');
+        code = code.replace('} from "@angular/core";', `${code.includes('repeated-block') ? ',': ''}forwardRef } from "@angular/core";`);
+      }
+      return code;
+    }
+  }
+});
+
+const ANGULAR_PLUGIN5 = () => ({
+  code: {
+    post: (code) => {
+      if (code.includes('component-ref, ComponentRef')) {
+        code = code.replace('ngOnInit() {\n', `ngOnInit() {\n  this.Wrapper = this.isInteractive ? InteractiveElement : this.componentRef;\n`)
+      }
+      return code;
+    }
+  }
+});
```

Comment:
Would appreciate a brief comment for each plugin, explaining what the workaround is and why it is needed.

Reply:
added